src/StateMachine/Applicator/MollieOrderStatesApplicator.php

```diff
@@ -57,21 +54,12 @@ public function execute(Order $order): void
 
         if (
             $order->isShipping() &&
-            $this->isConfirmNotify($order, $firstShipment) &&
-            false === $this->isShippingAllItems($firstShipment)
+            $this->isConfirmNotify($order, $firstShipment)
```

Comment:
Since we're removing partial shipments, everything in here should be based only on the last shipment, to be in line w/ Sylius default logic.